src/andromede/simulation/output_values.py

```diff
@@ -247,3 +247,107 @@ def _are_mappings_close(
             )
     else:
         return True
+
+
+@dataclass(frozen=True)
+class BendersSolution:
+    data: Dict[str, Any]
```

Comment:
Note that we could use `pydantic` to perform parsing + validation of the JSON content, while having a more typed class (with usual named fields etc).

Can be a future improvement, it's an implementation detail.

Reply:
I didn't know that, I will have a look later :)